### ISSUE TITLE
Add attribute to configure value property

### ISSUE THREAD

**Issue by staale** [cibernox/ember-power-select#481]
Say I have an array like this:
`[{id: 1, name: "name"}]`

Now, the actual selected value, will be the id, but what I want to display is the name. So I wish I could use power-select like this:
`{{#power-select keyField='id' options=options selected=selectedId onchange=(action (mut selectedId))}} as |option|}}
  {{option.name}}
{{/power-select}}`

To fix this, i currently have to make a custom computed property, that maps from the selected object to write the selected id internally, and return the selected object based on the selected id.

**Comment by cibernox** [comment]
You can
```
{{#power-select options=options selected=(find-by 'id' selectedId options) onchange=(action (mut selectedId) value="id")}} as |option|}} 
  {{option.name}} 
{{/power-select}}
```

**Comment by staale** [comment]
That works for single select, I will see if I can get it working for multiselect as well, however I fear that will be more complex.

**Comment by cibernox** [comment]
@staale For multiple selects you might need to use a computed property or create a `filter-by-collection` helper. 
This addon intentionally works always with full objects and not with properties of those objects. I understand that some usages might be a bit more verbose, but also makes the component much simpler inside.

Alternatively you could work on your app side with the full objects instead of with ids. I found that approach nicer. I only use the properties of the objects when I want to serialize them to the URL.

**Comment by cibernox** [comment]
One pure-helper idea:

```hbs
{{#power-select options=options selected=(filter-by 'id' (action "isAmong" selectedIds) options) ... as |option|}} 
  {{option.name}} 
{{/power-select}}
```

**Comment by drauschenbach** [comment]
Resolves issue #160
